docs/USERGUIDE.md

```diff
@@ -119,5 +119,5 @@ If you have registered a channel, you can make it private. The best way to do th
 
 1. Set your channel to be invite-only (`/mode #example +i`)
 1. Identify the users you want to be able to access the channel. Ensure that they have registered their accounts (you should be able to see their registration status if you `/WHOIS` their nicknames).
-1. Add the desired nick/account names to the invite exception list (`/mode #example +I alice`)
+1. Add the desired nick/account names to the invite exception list (`/mode #example +I alice`) or give them persistent voice (`/msg ChanServ AMODE +v alice`)
```

Comment:
I think this is a bit weird to have in two places, while it probably doesn't change that often, maybe it should be removed from MANUAL.md?

Reply:
A certain amount of duplication between the two documents is expected. MANUAL.md should be a comprehensive guide for ircops; USERGUIDE.md should be a short guide for end users. In this case, the information is relevant to end users (so it belongs in USERGUIDE.md) but also an obscure, Ergo-specific feature that operators should be aware of.